lib/SetAPI/sampleset/SampleSetInterface.py

```diff
@@ -37,6 +23,10 @@ def _check_condition_matching(self, conditionset_ref, matching_conditions):
             raise ValueError(error_msg)
 
     def create_sample_set(self, ctx, params):
+        # check the ws_id param to see if it is actually a ws_name to ensure
+        # we use the right form of workspace identifier when saving
+        ws_params = convert_workspace_param({"workspace": params.get("ws_id")})
+
```

Comment:
the following two lines actually overwrite input parameters -- presumably no one is using those input params as they would be wondering why they were never able to enter any sample IDs or conditions!

Reply:
🙀